src/router/index.js

```diff
@@ -0,0 +1,33 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+import Home from "../views/Home.vue";
+import NotFound from "../views/NotFound.vue";
+import Details from "../views/Details";
+
+Vue.use(VueRouter);
+
+const routes = [
+  {
+    path: "/",
+    name: "home",
```

Comment:
Since you're using router.push in the app specifying the route name, it's better to extract this detail into a config directory and use as a variable to avoid typos and promote the reuse.